app/assets/stylesheets/fae/modules/forms/_colorpicker.scss

```diff
@@ -1,9 +1,9 @@
-input.string.js-color-picker {
-  width: 180px;
+.js-color-picker {
+  width: 180px !important; // overrides input.string input
 }
 
 div.cp-color-picker {
```

Comment:
this 'over-qualifying selector' is unavoidable

Reply:
FWIW you can almost always get around `!important` in these situations with the specificity hack of doubling a selector. Here it'd look like `.js-color-picker.js-color-picker`, but that can be compared against the FINE standards of the moment (cc @aaronransley ). I'd personally favor doubling over an `!important`.

Reply:
+1 on the double selector hack. It's not pretty but beats an `!important` if possible. Thanks for the work on this feature @andrewdang17!